### ISSUE TITLE
A11y_AzureDataStudio_Connect to a SQL Server_Usability: Screen Reader is not announcing the status of a server is connected successfully or not

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/vscode#144588]
### GitHub Tags:
#A11yUsable; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG4.1.3; #Win32; #AINotInScope; #Status Messages; #Narrator; 

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Hover over the 'Servers' header and select the 'New Connection' icon.
5. Provide server and windows authentication details, then click 'Connect'.
6. Validate in left menu that server has been successfully connected.
7. Verify status messages after the server is connected successfully is announced by narrator or not .
### Note:
Similar issues can be observed in the following in the following:

1. Delete.
2. Conformation.

### Actual Result:
Screen Reader is not announcing the status of a server is connected successfully or not.

### Expected Result:
Screen Reader should be announced the status of a server is connected successfully .

### Attachment:
![A11yUsable_AzureDataStudio_Connect a SQL server](https://user-images.githubusercontent.com/99962579/157043865-dfb023e1-84f2-476c-ae54-c07ee849543f.gif)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
